src/Storages/ExternalStream/Kafka/Topic.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <boost/core/noncopyable.hpp>
+#include <librdkafka/rdkafka.h>
+
+namespace DB
+{
+
+namespace RdKafka
+{
+
+using Properties = std::vector<std::pair<std::string, std::string>>;
+
+class Topic : boost::noncopyable
+{
+public:
+    Topic(rd_kafka_t & rk, const std::string & name);
+    ~Topic() = default;
+
+    rd_kafka_topic_t * getHandle() const { return rkt.get(); }
+    std::string name() const { return rd_kafka_topic_name(rkt.get()); }
+    int describe() const;
```

Comment:
need a comment to explain the return value ?

Reply:
I updated the function name to make it more clear what it does.